client/my-sites/email/titan-add-mailboxes/titan-mailbox-pricing-notice.tsx

```diff
@@ -89,36 +100,44 @@ const TitanMailboxPricingNotice = ( { domain, titanMonthlyProduct } ) => {
 
 	const purchaseCost = getTitanMailboxPurchaseCost( domain );
 
-	if ( doesAdditionalPriceMatchStandardPrice( domain, titanMonthlyProduct ) ) {
+	if ( doesAdditionalPriceMatchStandardPrice( domain, titanMailProduct ) ) {
+		const translateArgs = {
+			args: {
+				price: purchaseCost.text,
+			},
+			components: {
+				strong: <strong />,
+			},
+			comment:
+				'%(price)s is a formatted price for an email subscription (e.g. $3.50, €3.75, or PLN 4.50)',
+		};
+
 		return (
 			<Notice icon="info-outline" showDismiss={ false } status="is-success">
-				{ translate(
-					'You can purchase new mailboxes at the regular price of {{strong}}%(price)s{{/strong}} per mailbox per month.',
-					{
-						args: {
-							price: purchaseCost.text,
-						},
-						components: {
-							strong: <strong />,
-						},
-						comment:
-							'%(price)s is a formatted price for an email subscription (e.g. $3.50, €3.75, or PLN 4.50)',
-					}
-				) }
+				{ isTitanMonthlyProduct( titanMailProduct )
+					? translate(
+						'You can purchase new mailboxes at the regular price of {{strong}}%(price)s{{/strong}} per mailbox per month.',
+						translateArgs
+					) : translate(
+						'You can purchase new mailboxes at the regular price of {{strong}}%(price)s{{/strong}} per mailbox per year.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'You can purchase new mailboxes at the regular price of {{strong}}%(price)s{{/strong}} per mailbox per month.' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/extra-professional-email-mailbox-purchase)